styles/styles.css

```diff
@@ -96,7 +96,7 @@ nav .nav-menu{
     text-align: center;
 }
 .section-1{
-    background-image: url('/images/bg-top.png');
+    background-image: url('../images/bg-top.jpg');
```

Comment:
Comments to styles.css
- line 6: invalid character at the end
- general: would be cool if you would sue grid or flex layout here, based on the task description that was allowed
- line 131, 144: don't need to repeat color and background here, you are not changing it on hover
- line 197: bold is a default for `h3`
- line 328, 340: I would rather use simpler selectors just by adding a classes to `input`